### PR TITLE
fix issue #185, pre-create manifests directory

### PR DESCRIPTION
**Pull request by anhowe**
This fixes issue #185.  This pre-creates manifests directory on manifest and agent in case there is ever a time where they do not have manifest files.

### CODE REVIEW

File: parts/kubernetesagentcustomdata.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ runcmd:
 - systemctl restart docker
 - /usr/bin/curl -sSL --retry 12 --retry-delay 10 https://storage.googleapis.com/kubernetes-release/release/{{{kubectlVersion}}}/bin/linux/amd64/kubectl > /usr/local/bin/kubectl
 - chmod +x /usr/local/bin/kubectl
+- /bin/mkdir -p /etc/kubernetes/manifests
```
Comment (colemickens) [inline parts/kubernetesagentcustomdata.yml]: Please use `mkdir`. If `/usr/bin` or `/bin` are never not-on-PATH, the system won't boot and we'll be so hosed it won't matter.

Comment (anhowe) [inline parts/kubernetesagentcustomdata.yml]: fixed

Comment (anhowe) [inline parts/kubernetesagentcustomdata.yml]: and deployed and confirmed working

**Comment by colemickens** [comment]
Go for it. I approved already.

**Comment by colemickens** [comment]
One more suggestion @anhowe. If your commit message or pull request text
says "Fixes #0" then the associated Issue is closed automatically with a
link to the PR.

As is now, you're closing without manually or automatically linking to the
PR so it's hard for people to discover where or how their issue was fixed.

**Pull request merged**